src/rethinkdb/query.cljc

```diff
@@ -223,9 +223,10 @@
   [table ids & [optargs]]
   (term :GET_ALL
         (cond
-         (sequential? ids) (concat [table] ids)
-         (map? ids)        [table (wrap-args ids)]
-         :else (throw (IllegalArgumentException. "get-all expects a collection or a map.")))
+          (sequential? ids) (concat [table] ids)
+          (map? ids)        [table (wrap-args ids)]
+          :else #?(:clj (throw (IllegalArgumentException. "get-all expects a collection or a map."))
+                   :cljs (throw (js/Error. "get-all expects a collection or a map."))))
```

Comment:
I have done it quickly yes, if you want I can change it no problem!

On May 9, 2016 2:05:53 AM PDT, Dan Kersten <notifications@github.com> wrote:
>> @@ -223,9 +223,10 @@
>>    [table ids & [optargs]]
>>    (term :GET_ALL
>>          (cond
>> -         (sequential? ids) (concat [table] ids)
>> -         (map? ids)        [table (wrap-args ids)]
>> -         :else (throw (IllegalArgumentException. "get-all expects a
>collection or a map.")))
>> +          (sequential? ids) (concat [table] ids)
>> +          (map? ids)        [table (wrap-args ids)]
>> +          :else #?(:clj (throw (IllegalArgumentException. "get-all
>expects a collection or a map."))
>> +                   :cljs (throw (js/Error. "get-all expects a
>collection or a map."))))
>
>Would something like this be better? 
>
>```
>:else (throw (#?@(:clj IllegalArgumentException. :cljs js/Error.)
>"get-all expects a collection or a map."))
>```
>
>(or `new #?@(...)` if the `Foo.` syntax doesn't work)
>
>Basically, to avoid duplicating the error message. Dunno if its worth
>it though.
>
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/apa512/clj-rethinkdb/pull/155/files/97447e828a932aab55915d26c4dc53f89b26d321#r62468975

-ar